.github/workflows/check-in-tree-build.yml

```diff
@@ -70,6 +70,17 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: llvm-project/llvm/projects/SPIRV-LLVM-Translator
+      - name:  Get tag for SPIR-V Headers
+        uses: juliangruber/read-file-action@v1
```

Comment:
Are we ok with using an [action](https://github.com/marketplace/actions/read-file) which is not certified by GitHub? Do we really need an extra step to read the file?

Reply:
Unfortunately, I wasn't able to find an alternative that would be as concise - retaining the tag in the separate file felt like the primary objective. Naturally, I'm open to any suggestions.

Reply:
UPD: Thanks for the suggestion, performed the switch in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/9ea401169f95972c7457e3e3fe5ef647cfc6ca8b.